src/client/styles/csp.ts

```diff
@@ -0,0 +1,17 @@
+const prod = process.env.NODE_ENV == 'production';
```

Comment:
Pra que serve esse arquivo?

Reply:
Essa primeira linha eu deixei para ver com vcs mesmo, não sei se precisa dela, o CSP é a meta tag do html que protege contra cross-site scripting, aí se for usar recurso de fora tem que adicionar o site de origem ai no csp. Se vier qualquer um injetar um script ai não aceita, talvez onde o next exiga 'unsafe inline' ali.. Recurso de segurança e melhora o SEO da página